README.md

```diff
@@ -392,6 +392,13 @@ Possible values are:
 
   https://en.wikipedia.org/wiki/Unix_time
 
+- `UnixTimeReadable`
```

Comment:
Other concerns about the reasonableness aside, `UnixTimeReadable` is a bit of a red herring of an option name.

It's not about the "readability" (`RfcTime` is readable for example), it's about the UTC.

Reply:
And the essence of Unix time is really that it's the seconds (or similar -- milliseconds or nanoseconds) count. If it's not that, it's not any kind of "Unix time" or epoch time.